src/monarch_gene_mapping/main.py

```diff
@@ -0,0 +1,80 @@
+from os import sep
+import typer
+import pathlib
+
+from kghub_downloader.download_utils import download_from_yaml
+from sssom.context import get_converter
+from curies import Converter, chain
+
+from monarch_gene_mapping.cli_utils import generate_gene_mappings
+from monarch_gene_mapping.uniprot_idmapping_preprocess import filter_uniprot_id_mapping_file
+
+typer_app = typer.Typer()
+sssom_converter = get_converter()
+HERE = pathlib.Path(__file__).parent.absolute()
+DOWNLOAD_YAML = f"{HERE}/download.yaml"
+
+prefixes = {
+    "HGNC": "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/",
+    "WB": "https://wormbase.org/",
+}
+gene_converter = Converter.from_prefix_map(prefixes)
+converter = chain([sssom_converter, gene_converter])
```

Comment:
Note that @kevinschaper has changed this code in gene mapping commons yesterday https://github.com/monarch-initiative/monarch-gene-mapping/pull/39

Reply:
I made the same changes locally, but the sssom parse line in the makefile fails

Reply:
do you have sssom version 0.4.2 installed in your environment?

Reply:
i do indeed. i've commmitted my changes , but the output is:
```
ValueError: {'ZFIN', 'FB', 'HGNC', 'UniProtKB', 'OMIM', 'RGD', 'Xenbase'} are used in the SSSOM mapping set but it does not exist in the prefix map
make[1]: *** [monarch_mapping_commons.Makefile:39: mappings/gene_mappings.sssom.tsv] Error 1
make[1]: Leaving directory '/mnt/dev/dev/TISLab/monarch-mapping-commons'
make: *** [Makefile:37: mappings] Error 2
```

Reply:
my guess: standardize_curies() previously standardised these prefixes to whatever is `sssom.context.get_coverter()`, which is probably some lower case thing, and since `sssom parse` relies on that converter, it does not know what to do with these prefixes.

Add the prefixes in `$(METADATA_DIR)/gene_mappings.sssom.yml` to circumvent this issue, but yea, this is all not at all ideal. Ideally the gene mapping pipeline writes its own sssom metadata header with all the right prefixes (you can just print `converter.bimap` if you want to get the list of prefixes, or even better, create a mappingsetdataframe with the SSSOM API with the new converter and then use sssom.writer.write_table()).

Reply:
added prefixes to metadata file